<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtClass</class>
 <widget class="QMainWindow" name="qtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ROI_Button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI</string>
    </property>
   </widget>
   <widget class="QLabel" name="a_image">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="X_Slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="X_Spin"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="Y_Slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Y_Spin"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>W</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="W_Slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="W_Spin"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="H_Slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="H_Spin"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="hist_button">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>HIST</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFLIE">
    <property name="title">
     <string>FLIE</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menussss">
    <property name="title">
     <string>ssss</string>
    </property>
   </widget>
   <addaction name="menuFLIE"/>
   <addaction name="menussss"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Open"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="Open">
   <property name="icon">
    <iconset>
     <normaloff>../folder.png</normaloff>../folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>qtClass</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Open</sender>
   <signal>triggered()</signal>
   <receiver>qtClass</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ROI_Button</sender>
   <signal>clicked(bool)</signal>
   <receiver>qtClass</receiver>
   <slot>ROI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>H_Slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>H_Spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>709</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>W_Slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>W_Spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>709</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Y_Slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>Y_Spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>709</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>X_Slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>X_Spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>710</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>H_Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>H_Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>637</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Y_Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Y_Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>637</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>X_Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>X_Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>637</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>W_Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>W_Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hist_button</sender>
   <signal>clicked()</signal>
   <receiver>qtClass</receiver>
   <slot>HIST()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>726</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>trigger()</slot>
  <slot>open()</slot>
  <slot>ROI()</slot>
  <slot>Slider()</slot>
  <slot>HIST()</slot>
 </slots>
</ui>
